src/en/readkomik/src/eu/kanade/tachiyomi/extension/en/readkomik/ReadKomik.kt

```diff
@@ -4,7 +4,13 @@ import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
 import eu.kanade.tachiyomi.network.interceptor.rateLimit
 import okhttp3.OkHttpClient
 
-class ReadKomik : MangaThemesia("Readkomik", "https://readkomik.com", "en") {
+class ReadKomik : MangaThemesia(
+    "Readkomik",
+    "https://rkreader.org",
+    "en",
+    "/archives/manga",
```

Comment:
does this work? 

this encode the middle "/" in the url, as MangaThemesia uses `addPathSegment`

Reply:
Does.

The default ```mangaUrlDirectory``` is ```/manga```, and ```substring(1)``` is used when ```addPathSegement``` is used.

Sample: https://github.com/keiyoushi/extensions-source/blob/1233aa8816188a887eb30dcd517371b39581e4bf/lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt#L93

Reply:
> does this work?
> 
> this encode the middle "/" in the url, as MangaThemesia uses `addPathSegment`

You're right! 

The first change wasn't enough, I noticed other problems caused by the / in the middle and made other specific improvements.

The first change worked, but it wasn't good